README.md

```diff
@@ -26,30 +27,30 @@ See the [`LICENSE`](./LICENSE) file for details.
 
 We would love involvement from more people! If you notice any errors or would like to submit changes, please see our [Contributing Guidelines](./.github/CONTRIBUTING.md).
 
-We use GitHub issues for [tracking bugs and feature requests](https://github.com/datatogether/webapp/issues) and Pull Requests (PRs) for [submitting changes](https://github.com/datatogether/webapp/pulls)
+We use GitHub issues for [tracking bugs and feature requests](https://github.com/datatogether/webapp/issues) and Pull Requests (PRs) for [submitting changes](https://github.com/datatogether/webapp/pulls).
 
 ## Installation
 
-To get started with dev, make sure you have [docker](https://store.docker.com/search?type=edition&offering=community) and [docker compose](https://docs.docker.com/compose/install/) And then run:
+To start developing, make sure you have [docker](https://store.docker.com/search?type=edition&offering=community) and [docker compose](https://docs.docker.com/compose/install/) And then run:
 
 ```shell
 $ git clone git@github.com:datatogether/webapp.git
 $ cd webapp
 $ docker-compose up
 ```
 
-This'll spin up a whole bunch of servers and make lots of noise. Once you see `webpack built` you can go to `http://localhost:3000` to see the app running locally.
+This will spin up a whole bunch of servers (most of the repositories in this organization are dependencies) and make lots of noise. Once you see `webpack built` you can go to `http://localhost:3000` to see the app running locally.
 
 ## Development
 
-Developing is as simple as editing files in source. If webpack can, it'll automatically recompile & update your browser window. Have the browser's console open to make sure nothing's exploding.
+Once docker has spun up, webpack will watch `.js`, `.scss`, amd `.css` files and attempt to automatically recompile & refresh your browser window on changes. It's a good idea to have the browser's console open to make sure nothing's exploding. 
```

Comment:
`docker` ➜ `Docker`

Errrm, the part about `make sure nothing's exploding` could maybe be a bit more specific, but I'm already being too nit-picky so we could just it go.